src/Pos/Statistics/Statsd.hs

```diff
@@ -0,0 +1,72 @@
+{-# LANGUAGE ApplicativeDo #-}
+
+module Pos.Statistics.Statsd
```

Comment:
Not critical, but a comment explaining what "Statsd" means would make things clealer.